tests/Doctrine/Tests/Common/Cache/PhpFileCacheTest.php

```diff
@@ -65,12 +65,25 @@ public function testImplementsSetState()
         $this->assertTrue($cache->contains('test_set_state'));
     }
 
-    public function testNotImplementsSetState()
+    /*public function testNotImplementsSetState()
```

Comment:
this should not be commented. If it is not relevant anymore, it should be removed. Keeping track of the history is done using Git, not using code comments (which suck at it anyway).